Packs/Base/Scripts/DBotShowClusteringModelInfo/DBotShowClusteringModelInfo.py

```diff
@@ -0,0 +1,75 @@
+from CommonServerPython import *
+
+# CONSTS
+BASE_INCIDENT_COLUMNS = ['id', 'created', 'name', 'Group name', 'Group id']
+
+
+def show_model_info(model_name):
+    success = False
+    start = datetime.now()
+    model = None
+
+    while not success and datetime.now() - start < timedelta(minutes=10):
+        res = demisto.executeCommand("getMLModel", {"modelName": model_name})
+        if is_error(res):
+            time.sleep(5)
+        else:
+            success = True
+            model = res[0]['Contents']
+
+    if model:
+        md = model['model']['extra']['modelSummaryMarkdown']
+        return_outputs(md)
+    else:
+        return_outputs("Cannot find model " + model_name)
+
+
+def show_incidents_in_cluster(model_name, query, display_fields):
+    res = demisto.executeCommand("DBotTrainClustering", {'modelName': model_name})
+    data = res[0]['Contents']['data']
+    id_to_cluster = {}
+    id_to_cluster_name = {}
+    incidents = []  # type: ignore
+    for row in data:
+        if row['pivot'] in query.split(" "):
+            incidents_in_cluster = json.loads(row['incidents'])
+            incidents += incidents_in_cluster
+            for inc in incidents_in_cluster:
+                id_to_cluster[inc['id']] = re.sub('\D', '', row['pivot'])
+                id_to_cluster_name[inc['id']] = row['name']
+
+    incidents_to_show = []
+    for inc in incidents:
+        incident_to_show = {k: v for k, v in inc.items() if k in display_fields}
+        incident_to_show['Id'] = "[{0}](#/Details/{0})".format(inc['id'])
+        incident_to_show['Name'] = incident_to_show['name'].replace("#", "")
+        incident_to_show['Created'] = incident_to_show['created'][:incident_to_show['created'].find(".")]
+        incident_to_show['Group id'] = id_to_cluster[inc['id']]
+        incident_to_show['Group name'] = id_to_cluster_name[inc['id']]
```

Comment:
Use **get** to avoid errors

Reply:
I will use get - but it's not possible for this to be not exists